packages/ckeditor5-dev-env/tests/index.js

```diff
@@ -133,6 +135,26 @@ describe( 'dev-env/index', () => {
 		} );
 	} );
 
+	describe( 'updatePackageVersions()', () => {
+		it( 'should update versions in package.json files', () => {
```

Comment:
If you could add some new tests it would be great, especially for the conditions you have in the script (i.e. the regex for matching the dependency names).

Reply:
49cb082